### PR TITLE
Adds faq acme debug

### PR DESCRIPTION
**Pull request by JoshVanL**
Signed-off-by: JoshVanL <vleeuwenjoshua@gmail.com>

/assign @munnerz

### CODE REVIEW

File: content/en/docs/configuration/acme/http01/_index.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ weight: 30
 type: "docs"
 ---
 
+TODO: add lets encrypt URLs to request from
```
Comment (munnerz) [inline content/en/docs/configuration/acme/http01/_index.md]: ? what's this?

File: content/en/docs/faq/acme.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
+title: "Troubleshooting Issuing ACME Certificates"
+linkTitle: "Troubleshooting Issuing ACME Certificates"
+weight: 60
+type: "docs"
+---
+
+When requesting ACME certificates, cert-manager will create `Order` and
+`Challenges` to complete the request. As such, there is more resources to
```
Comment (munnerz) [inline content/en/docs/faq/acme.md]: `there are more`

**Comment by JoshVanL** [comment]
back to you

**Pull request merged**